src/solvers/refinement/string_refinement.cpp

```diff
@@ -1793,7 +1795,7 @@ static std::pair<bool, std::vector<exprt>> check_axioms(
     const exprt with_concrete_arrays =
       substitute_array_access(negaxiom, gen_symbol, true);
 
-    stream << indent << i << ".\n";
+    stream << indent << i << '.' << eom;
```

Comment:
Just observing (but not fully understanding) that you use `messaget::eom` above while here you seem to be getting away with just `eom`. Maybe that's the case above as well?

Reply:
In this function we defined `const auto eom=messaget::eom;`, that's why we can use it here like that.